cicecore/drivers/nuopc/cmeps/ice_import_export.F90

```diff
@@ -913,6 +943,26 @@ subroutine ice_export( exportState, rc )
              ! ice fraction
              ailohi(i,j,iblk) = min(aice(i,j,iblk), c1)
 
+             ! ice thickness (m)
```

Comment:
A comment about a preceding line (933) containing the OMP directive. Do additional variables need to be made private for the added floediam calculation?

Reply:
Good catch. Looks like k and floe_rad_c need to be added to the OMP PRIVATE statement. Anything else you noticed?

Reply:
I had the wave-ice coupling working in a branch, but I need to make sure it works if we don't have wave-coupling activated. 

I also noticed the change at LN 802 in import export---is this related to the c-grid development?

Reply:
Yes! The C-grid has been merged onto main.

Reply:
Also, just found out that this code for the FSD needs to be in an if block. Probably the set_stateexport and so on needs to check for the fields being asked for as well.